Templates/AppSource App/.github/workflows/CICD.yaml

```diff
@@ -242,18 +243,30 @@ jobs:
           gitHubSecrets: ${{ toJson(secrets) }}
           getSecrets: '${{ steps.envName.outputs.envName }}-AuthContext,${{ steps.envName.outputs.envName }}_AuthContext,AuthContext,${{ steps.envName.outputs.envName }}-EnvironmentName,${{ steps.envName.outputs.envName }}_EnvironmentName,EnvironmentName,projects'
 
-      - name: Deploy
+      - name: Deploy AL
         id: Deploy
         uses: microsoft/AL-Go-Actions/Deploy@main
         env:
           Secrets: '${{ steps.ReadSecrets.outputs.Secrets }}'
         with:
           shell: powershell
           environmentName: ${{ matrix.environment }}
-          artifacts: '.artifacts'
+          artifactsFolder: '.artifacts'
           type: 'CD'
           deploymentEnvironmentsJson: ${{ needs.Initialization.outputs.deploymentEnvironmentsJson }}
 
+      - name: Deploy PP
+        if: env.type == 'PTE' && env.powerPlatformSolutionFolder != ''
+        uses: microsoft/AL-Go-Actions/DeployPowerPlatform@main
+        env:
+          Secrets: '${{ steps.ReadSecrets.outputs.Secrets }}'
+        with:
+          shell: powershell
+          actionsRepo: 'microsoft/AL-Go-Actions@main'
```

Comment:
This seems like there might be backwards compatibility issues.

Reply:
The actionsRepo line is modified by Update AL-Go System files exactly like other references to microsoft/AL-Go-Actions@main - meaning that it will point to the same repo as the action which we are calling - allowing that action to call other actions in the same version of AL-Go Actions.
This is the first action, which is a workflow, calling other actions - I could not find a better way to do this - if you have other ideas, I am open.